app/src/main/res/values-ar/voice_commands.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version='1.0' encoding='utf-8'?>
+<resources>
+  <string name="vChoice">اذكر الامر المطلوب</string>
+  <string-array name="voiceCommandsList">
```

Comment:
Avoid using camel case in the markup attributes, 'voiceCommandsList' should be 'voice_commands_list'.